modules/apps/asset/asset-list-service/src/main/java/com/liferay/asset/list/internal/asset/entry/provider/AssetListAssetEntryProviderImpl.java

```diff
@@ -411,34 +416,32 @@ private BooleanClause[] _getAssetCategoryIdsBooleanClauses(
 		};
 	}
 
-	private List<AssetListEntryAssetEntryRel> _getAssetListEntryAssetEntryRels(
-		AssetListEntry assetListEntry, long[] segmentsEntryIds) {
+	private Map<AssetListEntryAssetEntryRel, AssetEntry>
```

Comment:
The Map seems like overkill, don't we really only need the priority and groupIds?